python/dask_cudf/dask_cudf/core.py

```diff
@@ -753,14 +757,11 @@ def from_dask_dataframe(df):
     "rpow",
 ]:
     meth = getattr(cudf.DataFrame, name)
-    kwargs = {"original": cudf.DataFrame} if DASK_VERSION >= "2.11.1" else {}
-    DataFrame._bind_operator_method(name, meth, **kwargs)
+    DataFrame._bind_operator_method(name, meth, original=cudf.Series)
 
     meth = getattr(cudf.Series, name)
-    kwargs = {"original": cudf.Series} if DASK_VERSION >= "2.11.1" else {}
-    Series._bind_operator_method(name, meth, **kwargs)
+    Series._bind_operator_method(name, meth, original=cudf.Series)
 
 for name in ["lt", "gt", "le", "ge", "ne", "eq"]:
```

Comment:
```suggestion
for name in ("lt", "gt", "le", "ge", "ne", "eq"):
```

Since all we are doing here is looping, can we switch this to `tuple` instead of a `list`?